apps/etterlatte-behandling/src/main/kotlin/behandling/BrukerService.kt

```diff
@@ -49,7 +49,14 @@ class BrukerServiceImpl(
 
             else -> {
                 val geografiskTilknytning = tilknytning.geografiskTilknytning() ?: throw GeografiskTilknytningMangler()
-                norg2Klient.hentNavkontorForOmraade(geografiskTilknytning)
+                if (tilknytning.harBareLandTilknytning()) {
+                    if (tilknytning.land?.equals("NO") != null) {
+                        Navkontor(navn = "UKjent kontor, men har Norge satt", enhetNr = "ukjent enhetsnummer")
+                    }
+                    Navkontor(navn = "Nav utland - Ålesund", enhetNr = Enheter.AALESUND_UTLAND.enhetNr)
```

Comment:
Sikker på at denne er Ålesund utland og ikkje Utland?

Reply:
Umulig å si

Reply:
Eller vi kan vel slå opp mot en behandling som kanskje har info om det

Reply:
Switch her er vel mellom bostatt utland og utlandtilsnitt?

Men - det er vel ikke satt før vi kommer inn i behandling og det skjer etter vi har satt enhet?

Reply:
Ja det skal ha skjedd før det her, her henter vi bare navkontor for en opprettet sak.

Reply:
Men det er ikke hovedpoenget her, nå får man rød boks på alle som ikke er norge saker om man går på deres saksoversikt.

Reply:
Jeg slo opp utlandsenhetene 0001 og 4862 mot norg2 tjenesten for navkontor og begge returnerte en tom body, mao det finnes ikke et spesifisert navkontor for disse sakene.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3231/commits/fff4d99cf73dc851637c6ca38989709c7089c8c5